main.py

```diff
@@ -162,9 +163,8 @@ def main():
                                     'type': 'limit',
                                     'account': 'spot',
                                     'side': 'sell',
-                                    'iceberg': '0',
-                                    'price': last_price}
-                                logger.info('Sold coins:\r\n' + str(sold_coins[coin]))
+                                    'iceberg': '0'}
+                                logger.info('Sold coins:' + str(sold_coins[coin]))
```

Comment:
I don't know what the \r\n is for, and it bugged me, so I removed it, not important, take it or leave it.

Reply:
it's for a newline. Nowadays, both \r and \n are liable to each add a newline, instead of both being required on windows. Generally just \n is good enough (if a newline is desired).